cl/DrupalClient.js

```diff
@@ -3,7 +3,9 @@
  *   Contains the DrupalClient class.
  */
 
-import { DrupalBase } from "../shared/DrupalBase"
+import { DrupalBase } from '../shared/DrupalBase';
+
+const SERVICE_NAME = DrupalBase.SERVICE_NAME;
```

Comment:
Unify between uses of that const and "this.SERVICE_NAME" ?

Reply:
Done